seed/seed.go

```diff
@@ -205,6 +205,14 @@ type PreseedCapable interface {
 	LoadPreseedAssertion() (*asserts.Preseed, error)
 }
 
+// Copier can be implemented by a seed that supports copying itself to a given
+// destination.
+type Copier interface {
+	// Copy copies the seed to the given seed partition. The root parameter
+	// is treated as the root of the seed partition.
+	Copy(root string) error
```

Comment:
I think this should be UC20+  and take seedDir and label like Open below, label could be "" in which case the same label is reused

Reply:
Makes sense, will make that change.